spec/definitions/ExportSchedule.yaml

```diff
@@ -0,0 +1,103 @@
+type: object
+required:
+  - format
+  - name
+  - recurringInstruction
+  - scheduledStartDate
+properties:
+  id:
+    description: The export identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  status:
+    description: Status of export request
+    readOnly: true
+    type: string
+    enum:
+      - "pending"
+      - "processing"
+      - "completed"
+  name:
+    description: Export name
+    type: string
+  resource:
+    description: The type of resource being exported (eg., transaction).
+    type: string
+    readOnly: true
+  format:
+    description: Export result format
+    type: string
+    enum:
+      - "csv"
+  arguments:
+    description: Export request arguments that can be passed to filter and/or sort the result set
+    type: object
+    properties:
+      filter:
+        description: The collection items filter requires a special format. Use "," for multiple allowed values. Use ";" for multiple fields. See the filter guide for more options and examples about this format.
+        type: string
+      sort:
+        type: array
+        description: The collection items sort field and order (prefix with "-" for descending sort).
+        items:
+          type: string
+      q:
+        type: string
+        description: The partial search of the text fields.
+      criteria:
+        $ref: "#/definitions/Condition"
+  dateRange:
+    type: object
+    $ref: "#/definitions/DateRange"
+  emailNotification:
+    description: List of emails to be notified when export is completed
+    type: array
+    items:
+      type: string
+      format: email
+  recurringInstruction:
```

Comment:
Check this from the Google Calendar API:
```
recurrence[] | list | List of RRULE, EXRULE, RDATE and EXDATE lines for a recurring event, as specified in RFC5545. Note that DTSTART and DTEND lines are not allowed in this field; event start and end times are specified in the start and end fields. This field is omitted for single events or instances of recurring events.
-- | -- | --
```

Reply:
This will add quite a bit of complexity. Is this something that has to be supported in the initial version?

Reply:
By the way, it seems like EXRULE is deprecated